### PR TITLE
changing link to microsoft docs

### PR DESCRIPTION
**Pull request by sakthivetrivel**

### CODE REVIEW

File: cluster-autoscaler/README.md [hunk 1]
```diff
@@ -122,4 +122,4 @@ Right now it is possible to run Cluster Autoscaler on:
 * GCE https://kubernetes.io/docs/concepts/cluster-administration/cluster-management/
 * GKE https://cloud.google.com/container-engine/docs/cluster-autoscaler
 * AWS https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md
-* Azure https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/azure/README.md
+* Azure https://docs.microsoft.com/en-us/azure/aks/autoscaler
```
Comment (feiskyer) [inline cluster-autoscaler/README.md]: @sakthivetrivel Thanks. The new link is AKS only, while acs-engine also supports VMAS VMSS and ACS.

Could you update AKS part in file https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/azure/README.md instead?

**Comment by sakthivetrivel** [comment]
@feiskyer good catch, just changed it. thanks!

File: cluster-autoscaler/cloudprovider/azure/README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ kubectl create -f cluster-autoscaler-standard-msi.yaml
 
 **WARNING**: Cluster autoscaler depends on user provided deployment parameters to provision new nodes. It should be redeployed with new parameters after upgrading Kubernetes cluster (e.g. upgraded by `acs-engine upgrade` command), or else new nodes will be provisioned with old version.
 
-### AKS or ACS deployment
+### ACS deployment
```
Comment (feiskyer) [inline cluster-autoscaler/cloudprovider/azure/README.md]: Could you also remove `az aks list` below?

**Comment by sakthivetrivel** [comment]
When will we be able to get this merged?

**Pull request merged**